static/sass/styles.scss

```diff
@@ -16,6 +16,7 @@
 @import 'patterns_blog-post';
 @import 'patterns_blog-card';
 @import 'utility_crop';
+@import "pattern_takeovers";
```

Comment:
I think you might want to add pattern_engage (I think)? for the future?

Reply:
I didn't include that scss